scapy/packet.py

```diff
@@ -1284,16 +1285,22 @@ def getlayer(self,
             match = issubtype
         else:
             match = lambda cls1, cls2: bool(cls1 == cls2)
+        # Note:
+        # cls can be int, packet, str
+        # scls can be packet, str (packet or packet+field)
+        # ccls can be packet, str (packet only)
```

Comment:
I'm of course open to naming suggestions 😄

Reply:
What do you think of the follwing?
cls -> class_type
scls -> string_class_name
ccls -> class_name